src/governance/governanceTransactionsFactory.ts

```diff
@@ -0,0 +1,133 @@
+import { ArgSerializer, BigUIntValue, StringValue } from "../abi";
+import { Address, Transaction, TransactionsFactoryConfig } from "../core";
+import { GOVERNANCE_CONTRACT_ADDRESS_HEX } from "../core/constants";
+import { TransactionBuilder } from "../core/transactionBuilder";
+import {
+    ChangeConfigInput,
+    ClearEndedProposalsInput,
+    CloseProposalInput,
+    NewProposalInput,
+    VoteProposalInput,
+} from "./resources";
+
+interface IConfig {
+    chainID: string;
+    addressHrp: string;
+    minGasLimit: bigint;
+    gasLimitPerByte: bigint;
+    gasLimitForProposal: bigint;
+    gasLimitForVote: bigint;
+    gasLimitForClosingProposal: bigint;
+    gasLimitForClearProposals: bigint;
+    gasLimitForChangeConfig: bigint;
+    gasLimitForClaimAccumulatedFees: bigint;
+}
+
+const EXTRA_GAS_LIMIT_FOR_VOTING = 100_000n;
```

Comment:
Maybe in config, as well? Similar to e.g. `additionalGasLimitPerValidatorNode`.

Reply:
maybe, we'll think about it.